README.md

```diff
@@ -111,76 +111,85 @@ The configuration file must be json formatted and may contain all or some of the
 
 ### Requirements
 
-This adapter requires an Anchore Engine or Enterprise deployment to operate against. The adapter can be deployed before the anchore installation,
+This adapter requires both a Harbor deployment and an Anchore Engine or Enterprise deployment to operate against. The adapter can be deployed before the Anchore installation,
```

Comment:
I think at this point we should just remove the Anchore Engine reference, it's been in public archive for over a year and I'm not sure we can even claim to work against it anymore

Reply:
Updated